src/styles/header/header-page.scss

```
@@ -0,0 +1,49 @@
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.nav {
+  &__list {
+    display: flex;
+    padding: 23px 0;
+  }
+  &__item {
```

```suggestion
  }
  
  &__item {
```

Add spaces between rules